syscall_sdl.c

```diff
@@ -9,9 +9,75 @@
 
 #include "state.h"
 
+#define MAX_EVENT_QUEUE_SIZE 100
+#define SCREEN_SCALE 4
```

Comment:
Don't set `SCREEN_SCALE`, which is not always fitting.

Reply:
Should I just remove the screen upsampling feature or add an extra screen_scale parameter in the `syscall_draw_frame`(and `check_sdl`), or implement the screen upsampling in the game itself? The original 640x384 resolution of quake-embedded is too high and will cause the game become neither interactive nor playable, however, 320x200 is quite efficient, but it makes the window of the game too small in my opinion.

Reply:
The customization and/or optimization for Quake should be landed in another pull request. Since this pull request is intended for adding input specific system calls, no functional changes against `syscall_draw_frame` would appear.

Reply:
Sooner or later, we can add some hotkeys which dynamic scale SDL window at runtime though.